docs/building-a-form/available-components.md

```diff
@@ -204,7 +266,32 @@ This indicates to the user that they have either not filled out a required field
 
 #### Usage guidelines
 
-- ?
+Adding a field to the `required` property in `schema` will automatically render an error on the field if it is blank. There are [default error messages](/src/js/validations.js#L24) for different situations. If you want to display a custom error message, add the message you want to the `ui:errorMessages` object in the `uiSchema`. You must add the error message as a key value pair: the key is the `schema` property that the data is in violation of (e.g., the entry doesn't match the requirements of the `pattern` property), and the value is the text of the error message. You may have more than 1 message in the `ui:errorMessages` object; they will be evaluated in order.
```

Comment:
> Adding a field to the `required` property in `schema` will automatically render an error on the field if it is blank. There are [default error messages](/src/js/validations.js#L24) for different situations. If you want to display a custom error message, add the message you want to the `ui:errorMessages` object in the `uiSchema`. You must add the error message as a key value pair: the key is the `schema` property that the data is in violation of (e.g., the entry doesn't match the requirements of the `pattern` property), and the value is the text of the error message. You may have more than 1 message in the `ui:errorMessages` object; they will be evaluated in order.

How about:

If you add a blank field to the `required` property in `schema`, one of several [default error messages](/src/js/validations.js#L24) will render. To show a custom error message, add the message to the `ui:errorMessages` object in the `uiSchema` as a key value pair:

- The key is the `schema` property that the data is in violation of (e.g., the entry doesn't match the requirements of the `pattern` property)
- The value is the text of the error message.

When you include multiple messages in the `ui:errorMessages` object, they're evaluated in order.

Reply:
It's not that you add a blank field, but rather adding the name of the field to the `required` property will trigger an error if that field is blank when the user tries to go to the next page. I think I can rephrase that part slightly, but keep the rest of your wording.